@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/damage/fn_damageSystem.sqf

```diff
@@ -74,49 +74,24 @@ if (_system == "trans") then {
         [_heli, "apu"] execvm "\fza_ah64_controls\scripting\damage\enginefire.sqf";
     };
 };
-
-if (_system == "fuselage") then {
-    if (_damage == 0 && _oldDam != 0) then {
-        _heli setVariable ["fza_ah64_irjamfail", true, true];
-    };
+/*
+if (_system == "lfuel") then {
     if (_damage >= 0.5) then {
-        _rand = round(random 10);
-        if (_rand == 1 && !(_heli getVariable "fza_ah64_irjamfail")) then {
-            _heli setVariable ["fza_ah64_irjamfail", true, true];
-        };
+        _rand = round(random 5);
         if (_rand == 5) then {
-            [_heli, "fwd"] execvm "\fza_ah64_controls\scripting\damage\fuel_leak.sqf";
-        };
-        if (_rand == 10) then {
-            [_heli, "aft"] execvm "\fza_ah64_controls\scripting\damage\fuel_leak.sqf";
-        };
-    };
-    if (_damage >= 0.8) then {
-        _rand = round(random 10);
-        if (_rand > 3 && !(_heli getVariable "fza_ah64_irjamfail")) then {
-            _heli setVariable ["fza_ah64_irjamfail", true, true];
-        };
-        if (_rand > 6) then {
-            [_heli, "fwd"] execvm "\fza_ah64_controls\scripting\damage\fuel_leak.sqf";
-        };
-        if (_rand > 8) then {
-            [_heli, "aft"] execvm "\fza_ah64_controls\scripting\damage\fuel_leak.sqf";
+            [_heli, "fwd"] spawn fza_fnc_DamageFuelLeak;
```

Comment:
If we are not using this function, it is worth removing it to make sure we don't think it's being used